examples/scraping-zarf-agent/manifests/prometheus-operator/clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: scrape-cr
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/status
+  - endpoints
+  - services
+  verbs:
+  - watch
+  - get
+  - list
```

Comment:
The linter complains when the file doesn't end in a newline
![image](https://github.com/defenseunicorns/zarf/assets/3977569/217b554e-e51e-47f2-afe6-a527b2d67e38)